user-service/.env.sample

```diff
@@ -1,6 +1,7 @@
-USER_SERVICE_MONGODB_URI="maxhasntsendyet"
-BLIST_MONGODB_URI="notcreatedyet"
```

Comment:
confirm `BLIST_MONGODB_URI` not needed?

Reply:
he created a new table in user service instead

Reply:
yeah we choose to use 1 db instead